scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -862,8 +863,9 @@ export function AuthoringDirective(
             };
 
             $scope.sendToNextStage = function() {
-                var stageIndex, stageList = desks.deskStages[desks.activeDeskId];
-                var selectedStage, selectedDesk = desks.deskLookup[desks.activeDeskId];
+                var currentDeskId = desks.getCurrentDeskId() || (userDesks && userDesks[0]);
```

Comment:
What's the change here? Does something not work with using `activeDeskId`? Why is fallback to `userDesks[0]` used?

Reply:
I think `getCurrentDeskId` uses `activeDeskId` so it is better to use the `getCurrentDeskId` here. It also considers users desks as well, so I removed some of the redundant bits there. Also added an error message where there is no current desk id (esp if the user is working in his personal space).